frontend/src/components/MainPage/MainPage.js

```diff
@@ -1,6 +1,7 @@
 import React, { useEffect, useState } from "react";
 import MonthlyIncome from "../MonthlyIncome/MonthlyIncome";
 import Container from "react-bootstrap/Container";
+import AddTransaction from "../TransactionTable/AddTransaction";
```

Comment:
lgtm, nice adding to main page instead of transaction row

Reply:
ty!